probes/pinsDependencies/impl.go

```diff
@@ -71,46 +72,23 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 				e := sce.WithMessage(sce.ErrScorecardInternal, "empty File field")
 				return findings, Probe, e
 			}
-			f = f.WithMessage(*rr.Msg).WithOutcome(finding.OutcomeNotApplicable)
+			f = f.WithMessage(*rr.Msg).WithOutcome(finding.OutcomeNotSupported)
 			findings = append(findings, *f)
 			continue
 		}
 		if rr.Msg != nil {
```

Comment:
If `rr.Msg` being non-nil is some kind of exceptional state, I think its name should reflect this.

Reply:
`rr.Msg` is old (~2y) code and has the comment "// Only for debug messages". 

This was before `ProcessingErrors []ElementError` was a thing, so we could migrate away from the debug messages. That would be a separate PR in my opinion.